Automattic/wp-calypso
# Post Publish Preview: Fix z-index to allow Masterbar usage

Fixes #15717.

This PR adjusts the z-index of the editor-preview such that Masterbar usage is still possible.

Before:
![image](https://user-images.githubusercontent.com/363749/27749619-ed50f1c2-5d99-11e7-98e7-a63975202b46.png)

After:
![image](https://user-images.githubusercontent.com/363749/27749671-1909ea80-5d9a-11e7-9232-0f2db9827850.png)


To test:
* Checkout branch.
* `make run`.
* Ensure that you're in the abtest by entering the following in your console: `localStorage.setItem('ABTests','{"postPublishPreview_20170627":"showPostPublishPreview"}')`.
* Visit the editor, draft and publish a post.
* Verify that you're able to use both the notification toolbar and select a site from the "Write" button while the preview is still visible.
